test/libsolidity/SolidityExecutionFramework.h

```diff
@@ -62,6 +62,37 @@ class SolidityExecutionFramework: public solidity::test::ExecutionFramework
 	) override
 	{
 		bytes bytecode = multiSourceCompileContract(_sourceCode, _sourceName, _contractName, _libraryAddresses);
+
+		if (bytecode.size() > 0)
+		{
+			if (bytecode[0] == uint8_t(0xef)) // is EOF
```

Comment:
I know this is testing only, but
```suggestion
			if (bytecode[0] == uint8_t(0xef) && bytecode[1] == uint8_t(00)) // is EOF
```

Plus the `bytecode.size() > 3` makes more sense.

Maybe also add a TODO note to validate version at some point.